### PR TITLE
Set the PYTHONPATH for the tests using python

### PR DESCRIPTION
**Pull request by lisajulia**
Add argument 'ENVIRONMENT' to add_test_compareSeparateECLFiles and use it to set the PYTHONPATH for the tests using python

### CODE REVIEW

**Comment by lisajulia** [comment]
jenkins build this please

**Comment by lisajulia** [comment]
jenkins build this please

**Comment by lisajulia** [comment]
jenkins build this please

**Comment by lisajulia** [comment]
jenkins build this please

File: CMakeLists.txt [hunk 1]
```diff
@@ -762,3 +762,8 @@ endif()
 
 install(DIRECTORY doc/man1 DESTINATION ${CMAKE_INSTALL_MANDIR}
   FILES_MATCHING PATTERN "*.1")
+
+if(opm-common_EMBEDDED_PYTHON)
+    set_target_properties(flow PROPERTIES
+                        ENVIRONMENT "PYTHONPATH=${PROJECT_BINARY_DIR}/python:${opm-common_DIR}/python:$ENV{PYTHONPATH}")
```
Comment (akva2) [inline CMakeLists.txt:768]: this makes little sense. you cannot set the environment for an executable, only on a test.

Comment (lisajulia) [reply CMakeLists.txt:768]: ok, thanks! That explains why that has no effect.. I also tried setting the environment variable in the shell script that execute the failing tests: https://github.com/OPM/opm-simulators/pull/5306/commits/fd2daa286f720c3d02bf2db8569ed0618e4225a9, but that also did not work

Comment (akva2) [reply CMakeLists.txt:768]: yeah, i just checked it and that code does the intended job at least.

File: python/simulators/CMakeLists.txt [hunk 2]
```diff
@@ -80,6 +80,8 @@ find_file(PYTHON_INSTALL_PY install.py
 add_custom_target(copy_python ALL
     COMMAND ${PYTHON_EXECUTABLE} "${PYTHON_INSTALL_PY}"
         ${PROJECT_SOURCE_DIR}/python/opm ${PROJECT_BINARY_DIR}/python 0
+    COMMAND ${PYTHON_EXECUTABLE} "${PYTHON_INSTALL_PY}"
+        ${PROJECT_SOURCE_DIR}/python/opm_embedded ${PROJECT_BINARY_DIR}/python 0
```
Comment (blattms) [inline python/simulators/CMakeLists.txt:84]: This seems to fail as there is no opm_embedded on my system

**Comment by blattms** [comment]
jenkins build this please

**Comment by lisajulia** [comment]
> jenkins build this please

OK, together with PR 4017 from opm-common that works..

**Comment by lisajulia** [comment]
jenkins build this opm-common=4017 please

**Comment by lisajulia** [comment]
jenkins build this please

**Comment by lisajulia** [comment]
The pipeline on https://github.com/OPM/opm-common/pull/4017 actually runs through without this change, closing it

**Pull request closed without merging**